### PR TITLE
DMEERX-794 Validate Mettles Rule Sets

### PR DESCRIPTION
**Pull request by yuliumitre**
This PR added Mettles Rule sets. 
Test procedures:
1. Switch branch to prosrule in CRD, test-ehr and CDS-Library repo
2. To load those test data, run the following command in test-ehr:
gradle loadBundleResource
3. Run request generator, select patient ONE-METTLE

### CODE REVIEW

**Comment by lpdecelles** [comment]
I am getting closer. I received this error.

image

**Comment by lpdecelles** [comment]
From Yolanda: 

Yeah, that is the warnings I mentioned in the document. They are caused by those value sets defined by Mettles are missing "expansion".

DTR is reading the value sets from "expansion"

**Comment by lpdecelles** [comment]
@yuliumitre, are the value sets in this PR not in VSAC?

**Comment by yuliumitre** [comment]
Correct. Mettles defines bunches of local value sets which miss "expansion".  CRD/DTR defines one local value set - COPD and it has "expansion" defined.

**Comment by lpdecelles** [comment]
We might need to move those local value sets into VSAC. I am not sure we can have them in our repo. For now we can merge. I'll create another Jira ticket to verify this.

**Comment by yuliumitre** [comment]
Not sure it should be our responsibility to do that.

**Comment by lpdecelles** [comment]
> Not sure it should be our responsibility to do that.

I agree, but we should understand the licensing. I have created a Jira sub-task for this and assigned it to me. :-)

**Comment by yuliumitre** [comment]
If it is not VSAC value set, it should be okay to check in to GitHub, right? We just want to make sure any of those VSAC value sets are not checked in to our repo. I will go through Mettels value sets and validate it.

**Comment by lpdecelles** [comment]
> If it is not VSAC value set, it should be okay to check in to GitHub, right? We just want to make sure any of those VSAC value sets are not checked in to our repo. I will go through Mettle value sets and validate it.

It is the underlying code system we need to review.

File: Examples/Shared/R4/resources/ValueSet-R4-immobilization.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "id": "2.16.840.1.113883.17.4077.3.2006",
+    "meta": {
+      "versionId": "1",
+      "lastUpdated": "2020-12-10T17:27:10.934+05:30",
+      "source": "#dLfN1XZj92k3Xpvr"
+    },
+    "identifier": [
+      {
+        "system": "urn:ietf:rfc:3986",
+        "value": "2.16.840.1.113883.17.4077.3.2006"
```
Comment (lpdecelles) [inline Examples/Shared/R4/resources/ValueSet-R4-immobilization.json]: This is in VSAC.

https://vsac.nlm.nih.gov/valueset/2.16.840.1.113883.17.4077.3.2006/expansion/Latest

**Comment by yuliumitre** [comment]
@lpdecelles I have cleaned up old Mettles rules and removed those VSAC value sets.

**Comment by lpdecelles** [comment]
@yuliumitre I am seeing this error with the latest changes.
image

**Comment by yuliumitre** [comment]
@lpdecelles can you try to use "New Incognito Window"? Sometimes the browser cache certain information.

**Comment by lpdecelles** [comment]
Hello Yolanda, I am seeing this error using this branch.
image

**Comment by yuliumitre** [comment]
Do you see any error from CRD console?

**Comment by lpdecelles** [comment]
> Do you see any error from CRD console?

Yes, I can show you at the DRLS meeting.

**Pull request merged**